comp/trace/bundle_test.go

```diff
@@ -23,11 +23,15 @@ import (
 func TestBundleDependencies(t *testing.T) {
 	require.NoError(t, fx.ValidateApp(
 		// instantiate all of the core components, since this is not done
-		// automatically.
-		coreconfig.Module,
+		// automatically. Use fx.Invoke to make sure components are initialized
+		// and all the providers are called.
 		fx.Supply(coreconfig.Params{}),
+		fx.Invoke(func(_ coreconfig.Component) {}),
+		coreconfig.Module,
+		fx.Invoke(func(_ config.Component) {}),
```

Comment:
Is it expected `Bundle` doesn't contain `config.Module`?

Reply:
@ogaca-dd I checked the original code from @truthbk and it contained it. Probably it went missing during all the tests changing the code to fit in it.

Reply:
Anyway, I updated the PR with this and all the comments. Please check.